standalone-metastore/src/main/thrift/hive_metastore.thrift

```diff
@@ -859,6 +859,15 @@ struct CurrentNotificationEventId {
     1: required i64 eventId,
 }
 
+struct NotificationEventsCountRequest {
+    1: required i64 fromEventId,
+    2: required string dbName,
+}
+
+struct NotificationEventsCount {
```

Comment:
NotificationEventsCountResponse would be more in line with naming used for most other methods.